### ISSUE TITLE
ArgumentException "element with same key was already added" when using in Parallel.ForEach

### ISSUE THREAD

**Issue by AndreR** [ExcelDataReader/ExcelDataReader#194]
Hello,

I'm using this library to load seven files in parallel. However, from time to time I get the error message in the title of this issue.

Here's the stack trace:

>    bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei Excel.Log.LogManager.Log(String objectName) in c:\development\ExcelDataReader\Excel\Log\LogManager.cs:Zeile 48.
   bei Excel.Log.LogManager.Log[T](T type) in c:\development\ExcelDataReader\Excel\Log\LogManager.cs:Zeile 28.
   bei Excel.Core.ZipWorker.NewTempPath() in c:\development\ExcelDataReader\Excel\Core\ZipWorker.cs:Zeile 248.
   bei Excel.Core.ZipWorker.Extract(Stream fileStream) in c:\development\ExcelDataReader\Excel\Core\ZipWorker.cs:Zeile 84.
   bei Excel.ExcelOpenXmlReader.Initialize(Stream fileStream) in c:\development\ExcelDataReader\Excel\ExcelOpenXmlReader.cs:Zeile 333.
   bei Excel.ExcelReaderFactory.CreateOpenXmlReader(Stream fileStream) in c:\development\ExcelDataReader\Excel\ExcelReaderFactory.cs:Zeile 74.
   bei de.Lechner.Statistik.API.FehlerController._Closure$__._Lambda$__17-3(String Path) in C:\Dev\Projekte\Lechner Statistik\de.Lechner.Statistik\de.Lechner.Statistik-Fehlerauswertung\de.Lechner.Statistik\Controllers\FehlerController.vb:Zeile 596.
   bei System.Threading.Tasks.Parallel.<>c__DisplayClass30_0`2.<ForEachWorker>b__0(Int32 i)
   bei System.Threading.Tasks.Parallel.<>c__DisplayClass17_0`1.<ForWorker>b__1()

The error appears in this line:
`Using reader = Excel.ExcelReaderFactory.CreateOpenXmlReader(str)`

**Comment by appel1** [comment]
Possibly a duplicate of #122.

**Comment by AndreR** [comment]
Looks like the other ticket was already closed. Is it possible to download a version with the fix already included somewhere?

**Comment by andersnm** [comment]
@AndreR The latest prealpha NuGets include this fix.
